packages/packages/commonwell-cert-runner/src/link-management.ts

```diff
@@ -7,8 +7,8 @@ import { patient, identifier, personStrongId } from "./payloads";
 // https://commonwellalliance.sharepoint.com/sites/ServiceAdopter/SitePages/Patient-Management-(PIX,-REST).aspx
 
 export async function linkManagement(commonWell: CommonWell, queryMeta: RequestMetadata) {
-  // C5: ​Levels of Link Assurance
-  console.log(`>>> ​C5a : Link a Patient to a Person upgrading from LOLA 1 to LOLA 2.`);
+  // C5: Levels of Link Assurance
+  console.log(`>>> C5a : Link a Patient to a Person upgrading from LOLA 1 to LOLA 2.`);
```

Comment:
These changes where Github doesn't indicate exactly what changed are related to invalid whitespaces (lint rule `no-irregular-whitespace`).